cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2895,9 +2910,7 @@ function start-kube-addons {
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
   local -r dst_dir="/etc/kubernetes/addons"
 
-  if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-    create-kubeconfig "addon-manager" ${ADDON_MANAGER_TOKEN} "${PROXY_RESERVED_IP}" "8888" "http"
-  elif [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
```

Comment:
ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE?

Reply:
will consider. this new variable has been discussed a couple time months ago. probably it is the time to get it in to getter refined controller of the flow.

Reply:
fixed as commented above